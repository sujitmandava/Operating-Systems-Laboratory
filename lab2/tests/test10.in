# Operation: -M
#Matrix multiplication success
3 3
1, 2,3 ,
6, 7,8,
11,12, 13,

3 5
15, 12,13 ,14, 15,
16, 17,18,19, 110,
111,112, 113,114, 115 ,