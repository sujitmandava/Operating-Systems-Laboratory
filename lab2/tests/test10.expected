3 5
380, 382, 388, 394, 580, 
1090, 1087, 1108, 1129, 1780, 
1800, 1792, 1828, 1864, 2980, 